src/main/java/com/iexec/resultproxy/chain/ChainConfig.java

```diff
@@ -16,20 +16,43 @@
 
 package com.iexec.resultproxy.chain;
 
+import com.iexec.commons.poco.chain.validation.ValidNonZeroEthereumAddress;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
 import lombok.Value;
+import org.hibernate.validator.constraints.URL;
+import org.hibernate.validator.constraints.time.DurationMax;
+import org.hibernate.validator.constraints.time.DurationMin;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 import java.time.Duration;
 
 @Value
 @ConfigurationProperties(prefix = "chain")
```

Comment:
You forgot an important annotation there
```suggestion
@Validated
@ConfigurationProperties(prefix = "chain")
```